SPECS/libvirt/libvirt.spec

```diff
@@ -27,6 +27,8 @@ BuildRequires:  cyrus-sasl-devel
 BuildRequires:  dbus-devel
 BuildRequires:  device-mapper-devel
 BuildRequires:  e2fsprogs-devel
+BuildRequires:  glusterfs-api-devel >= 3.4.1
```

Comment:
Could you add a comment explaining why this specific version?

Reply:
No idea, spec taken from, they didn't bother explaining but it doesn't hurt to keep it.